### ISSUE TITLE
Crash related to attributes

### ISSUE THREAD

**Issue by acedened** [tid-kijyun/Kanna#115]
### Installation method:
- [x] CocoaPods(1.1.0 or later)
### Library version:
- [x] v2.0.0
### Xcode version:
- [x] 8.0 (Swift 3.0) 

I don't know how to describe this problem, so I just created [MCVE](https://gist.github.com/acedened/3a46775beac79a151981ba56035c7626)

<img width="848" alt="2016-10-21 17 03 09" src="https://cloud.githubusercontent.com/assets/7037300/19601051/477a4258-97b0-11e6-969a-c1029a584868.png">

**Comment by acedened** [comment]
Note: may need multiple launches to reproduce.

**Comment by tid-kijyun** [comment]
Thank you for good issue report.
This problem occurs when the HTMLDocument destroys. 

```
if let doc = HTML(html: html, encoding: .utf8) {
    for elem in doc.body!.xpath("*") {
        elms.append(elem)
    }
}  // The doc is destroyed...

// Please don't access to any elements.
```

I want to fix this issue. Let me think about it.

Thanks

**Comment by droidery** [comment]
@tid-kijyun Can I store the `doc` variable and access it multiple times?

**Comment by tid-kijyun** [comment]
This issue was solved with the new release(v2.2.0).

```
var elems: [XMLElement] = []
if let doc = HTML(html: html, encoding: .utf8) {
    for elem in doc.body!.xpath("*") {
        elms.append(elem)
    }
}  // The doc is destroyed...

// You can access elements.
for elem in elems {
    print(elem)
}
```
